locales/en.default.schema.json

```diff
@@ -1983,6 +1983,187 @@
       "presets": {
         "name": "Video"
       }
+    },
+    "slideshow": {
+      "name": "Slideshow",
+      "settings": {
+        "layout": {
+          "label": "Layout",
+          "options__1": {
+            "label": "Full width"
+          },
+          "options__2": {
+            "label": "Grid"
+          },
+          "options__3": {
+            "label": "Grid with a peek"
```

Comment:
@katycobb Another copy I wanted to get your feedback on.

This is a setting to set the Layout. So it goes to:

- Layout
   - Full width
   - Grid
   - Grid with a peek

Reply:
@melissaperreault I'd suggest: 

Layout
- Full width
- Grid
- Grid with preview

Let me know if you are aligned!